### DISCUSSION TITLE
NGINX snippets not allowed

### DISCUSSION THREAD

**Discussion by jplspotter** [terrakube-io/terrakube#818] [General]
Hi,

In a kind cluster, the helm chart requires this step to allow the ingress snippets. It should be added to the documentation or remove the snippets from the nginx ingress if possible.

kubectl delete -A ValidatingWebhookConfiguration ingress-nginx-admission

**Comment by alfespa17** [comment]
Hello @jplspotter there is an open issue for that [here ](https://github.com/AzBuilder/terrakube-helm-chart/issues/90), maybe in the future we can remove the snipshet configuration if we find an alternative.

This [issue](https://github.com/AzBuilder/terrakube/issues/618) was also related

**Comment by tszebeni** [comment]
I have run into the same issue and enabled the config, redeployment went fine but I'm still unable to acces it the ui service in browser when I tested it in minikube.
I also deployed the helm chart into a GCP cluster and seems there are problems with ingress configuration: ui and dex are served well, but api requests are sent to ui service instead by ingress controller. Please recommend how to fix

**Reply by alfespa17** [reply]
Hello @tszebeni 

Did you update your local DNS settings when you deployed it in minikube as explained [here](https://docs.terrakube.io/getting-started/deployment/minikube#setup-dns-records)??

Maybe you can share some logs or the helm chart values for gcp that you are using in the deployment (without any sensitive value) so I can check what could be your issue
